### ISSUE TITLE
Unsupported site (submeta.io), some downloads work, others throw 403 forbidden error

### ISSUE THREAD

**Issue by PaperCloud10** [yt-dlp/yt-dlp#7113]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

United States

### Example URLs

Working: https://submeta.io/@lachlangiles/courses/omoplata/52cb94bd-95c2-4275-8b94-9cb003b0b6d3
Request URL: https://106vod-adaptive.akamaized.net/exp=1684918758~acl=/6696b647-e73a-440b-be2f-3859dd92bee9/*~hmac=78f56398cf8e687a35185f7901cf09a89cdd2aeb6b955b8f5c7e51da0fbfec31/6696b647-e73a-440b-be2f-3859dd92bee9/sep/video/1be0c6c0,380198e3,83919da9,ceffecf9,d09e2824/audio/7397cbaf,bff7aa2a,fcca82a0/master.mpd?base64_init=1&query_string_ranges=1

Not Working: https://submeta.io/@lachlangiles/courses/dlr-for-berimbolo/503ff6a0-b081-4c49-95af-bf7678c4ebd3
Request URL: https://9vod-adaptive.akamaized.net/exp=1684918177~acl=/7dd257c5-7d82-4f20-b273-8412cc1c7d08/*~hmac=1121a74f77f434540d8f308b78c2a943521448c14710684c81685a47dab4b6b2/7dd257c5-7d82-4f20-b273-8412cc1c7d08/sep/video/8cb91de5,a5f0395b,acba8ee9,e32c6c28,ebe16871/audio/3ea8618e,e59ed725/master.mpd?base64_init=1

### Provide a description that is worded well enough to be understood

I have been successfully using yt-dlp to download videos from other courses on this website, but a few courses have been throwing me this error. I have added all the request headers into my config file. The URL given to yt-dlp is the request URL found under the Network tab. For the request URL to donwload successfully, **I have to replace master.json with master.mpd**. 

Account details as follows:
User: leonchen
Pass: eh48cutpwt92

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-a', '/Users/leon/.config/yt-dlp/URLs.txt', '-vU']
[debug] User config "/Users/leon/.config/yt-dlp/config.txt": ['-o', '/Users/leon/Downloads/%(autonumber)02d.s%(epoch)s%', '-f', 'm4a+mp4', '--cookies-from-browser', 'brave', '--no-mtime', '--add-header', 'Accept:*/*', '--add-header', 'Accept-Encoding:gzip, deflate, br', '--add-header', 'Accept-Language:en-US,en;q=0.5', '--add-header', 'Origin:https://player.vimeo.com', '--add-header', 'Referer:https://player.vimeo.com/', '--add-header', 'Connection:keep-alive', '--add-header', 'sec-ch-ua:Chromium;v=112, Brave;v=112, Not:A-Brand;v=99', '--add-header', 'sec-ch-ua-mobile:?0', '--add-header', 'sec-ch-ua-platform:macOS', '--add-header', 'Sec-Fetch-Dest:empty', '--add-header', 'Sec-Fetch-Mode:cors', '--add-header', 'Sec-Fetch-Site:cross-site', '--add-header', 'Sec-GPC:1', '--user-agent', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36']
[debug] Batch file urls: ['https://9vod-adaptive.akamaized.net/exp=1684918177~acl=/7dd257c5-7d82-4f20-b273-8412cc1c7d08/*~hmac=1121a74f77f434540d8f308b78c2a943521448c14710684c81685a47dab4b6b2/7dd257c5-7d82-4f20-b273-8412cc1c7d08/sep/video/8cb91de5,a5f0395b,acba8ee9,e32c6c28,ebe16871/audio/3ea8618e,e59ed725/master.mpd?base64_init=1']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (pip)
[debug] Python 3.11.3 (CPython arm64 64bit) - macOS-12.6.3-arm64-arm-64bit (OpenSSL 1.1.1t  7 Feb 2023)
[debug] exe versions: ffmpeg 5.1.2 (setts), ffprobe 5.1.2
[debug] Optional libraries: Cryptodome-3.17, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from brave
[debug] Extracting cookies from: "/Users/leon/Library/Application Support/BraveSoftware/Brave-Browser/Default/Cookies"
[debug] using find-generic-password to obtain password from OSX keychain
[Cookies] Extracted 3252 cookies from brave
[debug] cookie version breakdown: {'v10': 3056, 'other': 0, 'unencrypted': 196}
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
yt-dlp is up to date (stable@2023.03.04)
[generic] Extracting URL: https://9vod-adaptive.akamaized.net/exp=1684918177~acl=/7dd257c5-7d82-4f20-b273-8412cc1c7d08/*~hmac=1121a74f77f434540d8f308b78c2a943521448c14710684c81685a47dab4b6b2/7dd257c5-7d82-4f20-b273-8412cc1c7d08/sep/video/8cb91de5,a5f0395b,acba8ee9,e32c6c28,ebe16871/audio/3ea8618e,e59ed725/master.mpd?base64_init=1
[generic] master: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] master: Extracting information
[debug] Identified a DASH manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] master: Downloading 1 format(s): audio-3ea8618e+video-acba8ee9
[debug] Invoking http downloader on "https://9vod-adaptive.akamaized.net/exp=1684918177~acl=/7dd257c5-7d82-4f20-b273-8412cc1c7d08/*~hmac=1121a74f77f434540d8f308b78c2a943521448c14710684c81685a47dab4b6b2/7dd257c5-7d82-4f20-b273-8412cc1c7d08/sep/video/8cb91de5,a5f0395b,acba8ee9,e32c6c28,ebe16871/audio/3ea8618e,e59ed725/../../../../../parcel/audio/3ea8618e.mp4?r=dXM%3D"
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3229, in process_info
    partial_success, real_download = self.dl(fname, new_info)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 2970, in dl
    return fd.download(name, new_info, subtitle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/downloader/common.py", line 444, in download
    ret = self.real_download(filename, info_dict)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/downloader/http.py", line 376, in real_download
    establish_connection()
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/downloader/http.py", line 129, in establish_connection
    ctx.data = self.ydl.urlopen(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3742, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 525, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 634, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 557, in error
    result = self._call_chain(*args)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 749, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 525, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 634, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 563, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by october262** [comment]
i think most of his videos are on vimeo.com - https://vimeo.com/496708627

**Comment by pukkandan** [comment]
Same underlying issue as https://github.com/yt-dlp/yt-dlp/issues/6526

**Comment by shaheerszm** [comment]
still not working with `yt-dlp 2023.10.13`

`[debug] Command-line config: ['https://submeta.io/@talgatilyasov/courses/wrestling-foundations-i-offense/5478f042-bdc1-4925-a321-1eacb3914b29', '-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (debian*)
[debug] Python 3.11.4 (CPython x86_64 64bit) - Linux-5.15.90.1-microsoft-standard-WSL2-x86_64-with-glibc2.37 (OpenSSL 3.0.8 7 Feb 2023, glibc 2.37)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, pyxattr-0.8.0, secretstorage-3.3.3, sqlite3-3.40.1, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1890 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.10.13, Current version: stable@2023.10.13
yt-dlp is up to date (stable@2023.10.13)
[generic] Extracting URL: https://submeta.io/@talgatilyasov/courses/wrestling-foundations-i-offense/5478f042-bdc1-4925-a321-1eacb3914b29
[generic] 5478f042-bdc1-4925-a321-1eacb3914b29: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] 5478f042-bdc1-4925-a321-1eacb3914b29: Extracting information
[debug] Looking for embeds
ERROR: Unsupported URL: https://submeta.io/@talgatilyasov/courses/wrestling-foundations-i-offense/5478f042-bdc1-4925-a321-1eacb3914b29
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/yt_dlp/YoutubeDL.py", line 1567, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yt_dlp/YoutubeDL.py", line 1702, in __extract_info
    ie_result = ie.extract(url)
                ^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/generic.py", line 2575, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://submeta.io/@talgatilyasov/courses/wrestling-foundations-i-offense/5478f042-bdc1-4925-a321-1eacb3914b29`

**Comment by smnbajwa** [comment]
I made this to download from submeta
https://github.com/smnbajwa/submeta-dl
